std_msgs/Header header
float32 watt_hours
float32 watt_hours_charged